devto/utilities.go

```diff
@@ -1,10 +1,30 @@
 package devto
 
 import (
+	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 )
 
+func non2xx(res *http.Response) bool { return res.StatusCode/100 != 2 }
```

Comment:
What about adding verbosity to the name?, I feel like `non2xx` by itself is not describing what the method does.

Reply:
Sounds good to me. Would `is2xxResponseStatus` be good? I'd say `isErrorStatus`, but I feel like 1xx and 3xx aren't really error statuses the way 4xx and 5xx are

Reply:
I think we can go for `NonSuccesfulResponse` or something like that, I will leave it up to you, as soon as it is quite clear what the method is doing.